packages/flutter/lib/src/widgets/overlay.dart

```diff
@@ -1821,6 +1821,10 @@ class OverlayPortal extends StatefulWidget {
 
   final bool _targetRootOverlay;
 
+  /// Used in testing to check if the OverlayPortal is built with [OverlayPortal.targetsRootOverlay].
+  @visibleForTesting
+  static bool overlayPortalTargetIsRootOverlay(OverlayPortal portal) => portal._targetRootOverlay;
```

Comment:
I'd prefer we don't expose convenience methods for testing in `package/flutter`.